### PR TITLE
tests: skip my lang in i18n spread test

### PR DESCRIPTION
**Pull request by sergiocazzolato**
this is to avoid panic caused by error in the translations.

### CODE REVIEW

File: tests/main/i18n/task.yaml [hunk 1]
```diff
@@ -36,4 +36,4 @@ execute: |
     fi
 
     # shellcheck disable=SC2002
-    grep -v -E '(pt|ko|nl)_' /usr/share/i18n/SUPPORTED | xargs -d '\n' -Iloc env SNAPD_DEBUG=1 LANG=loc snap changes everything 2>&1 > /dev/null | NOMATCH panic:
+    grep -v -E '(pt|ko|nl|my)_' /usr/share/i18n/SUPPORTED | xargs -d '\n' -Iloc env SNAPD_DEBUG=1 LANG=loc snap changes everything 2>&1 > /dev/null | NOMATCH panic:
```
Comment (Meulengracht) [inline tests/main/i18n/task.yaml:39]: this file is missing the details section

File: tests/main/i18n/task.yaml [hunk 2]
```diff
@@ -1,5 +1,9 @@
 summary: Test that i18n works
 
+details: |
+  This test cheks the translations do not causes crashes nor warnings.
```
Comment (ernestl) [inline tests/main/i18n/task.yaml:4]: cheks => checks
the => that
causes => cause
crashes nor warnings => crashes or warnings

File: tests/main/i18n/task.yaml [hunk 3]
```diff
@@ -1,5 +1,9 @@
...
+  Checks that there are no panics when lancuage packs are installed.
```
Comment (ernestl) [inline tests/main/i18n/task.yaml:5]: lancuage => language

Perhaps one line is sufficient? Proposal:
"Checks that there are no translation related panics with language packs installed."

**Review by ernestl — CHANGES_REQUESTED** [review]
Details needs some tweaks, added proposal

File: tests/main/i18n/task.yaml (hunk 2 above)
Comment (sergiocazzolato) [reply tests/main/i18n/task.yaml:4]: updated

File: tests/main/i18n/task.yaml (hunk 3 above)
Comment (sergiocazzolato) [reply tests/main/i18n/task.yaml:5]: done

File: tests/main/i18n/task.yaml (hunk 2 above)
Comment (ernestl) [reply tests/main/i18n/task.yaml]: Thanks!

File: tests/main/i18n/task.yaml (hunk 1 above)
Comment (ernestl) [reply tests/main/i18n/task.yaml:42]: Added in the meantime

**Pull request merged**